tests/Froxlor/ValidateTest.php

```diff
@@ -86,12 +86,29 @@ public function testValidateIpCidr()
 		$this->assertEquals("12.34.56.78/24", $result);
 	}
 
+    public function testValidateIpv6Disallowed()
+    {
+        $this->expectException("Exception");
+        $this->expectExceptionCode(400);
+        $result = Validate::validate_ip2("2620:0:2d0:200::7/32", false, 'invalidip', false, false, true, true);
```

Comment:
you don't need `$result =` if you expect an exception

Reply:
Will remove it...